### ISSUE TITLE
Cannot add family to proceed

### ISSUE THREAD

**Issue by SandroWissmann** [Banderi/Ozymandias#12]
So I have the same issue on windows and on linux.

When I go to the main menu and reach the familiy menu I cannot Add a family:

file:///tmp/Spectacle.CRmpjV/Screenshot_20220921_164048.png

If I confirm this dialog the list stays empty. Maybe I do something wrong here.

I thought this was maybe a linux only issue because it was mentioned that something with the file pathes is not yet ported well.

**Comment by SandroWissmann** [comment]
And another question. Can it be in general that the menus are not yet right? For example when I start Pharaoh / Cleopatra I do not see an options menu at all.

**Comment by SandroWissmann** [comment]
I took a look and it looks like original Pharaoh creates under the root folder of its installation a /save folder which contains familiy name

![Screenshot_20220921_173110](https://user-images.githubusercontent.com/42449992/191547051-b460a68d-27ab-43be-babf-cd1c767c6bf8.png)

**Comment by SandroWissmann** [comment]
Actually there was already an empty save folder. Maybe problem is really create these family folder and the .dat file.

**Comment by SandroWissmann** [comment]
I can confirm you go into Pharaoh and add one family and then you can continue. On linux though it crashes then in the start campaign. I will invest some time to take a look at it. Also will check if I come more far in windows.

**Comment by SandroWissmann** [comment]
On Windows I can go all the way into the game itself. I'm very impressed what you could already all do there.

When I came in the game it was running very lagy on windows.

I will try to investigate what is going on with this menu stuff that works not all correctly.

**Comment by Banderi** [comment]
Yes, the family creation is not complete atm! In fact, saving anything is not yet functioning, be it player data, scores, savegames, maps etc. (apart from engine settings that worked in Augustus already I guess, and the initial prompt for the game files directory). I'm relying entirely on files that are present from an original game playthrough for now!

**Comment by Banderi** [comment]
> And another question. Can it be in general that the menus are not yet right? For example when I start Pharaoh / Cleopatra I do not see an options menu at all.

Yes, some menus have been slightly modified/added/tweaked/etc.; it's not a perfect 100% replication as Julius was to C3, more of a tweaked/enhanced version like Augustus. Some menus in the original game are very badly aligned for example, and I wanted to fix that!
